devtools/jsonapi/commands/run.go

```diff
@@ -0,0 +1,149 @@
+package commands
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/devtools/jsonapi"
+	"github.com/orbs-network/orbs-network-go/test/crypto/keys"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+func ShowUsage() {
```

Comment:
It should be done through `flag`. Same for any other place with `ShowUsage()`.

Reply:
What do you mean exactly ? you want the default "help" as it is provided by the flag library ?
Their help message revolves a lot around just what you can do with the flags themselfs , not the command line tool that you're building.
`ShowUsage()` actually shows what is the usage of the cli itself or the sub command.

The flags are merely a small part of it.

In addition, the flag library shows many system flags related to the testing framework which I don't really want people to see in their help as it's too noisy. 

I don't quite agree the the `flag` library is the way to go here.

Reply:
`flag` library won't show you anything about tests in your final product. We can keep a short example how to use the CLI but ultimately we need to list and show all flags so users can explore the tool themselves.